.busted

```diff
@@ -0,0 +1,12 @@
+return {
+    _all = {
+        coverage = false,
+        lpath = "lua/?.lua;lua/?/init.lua",
```

Comment:
nit: should we also set `lua = "nlua"` here?

Reply:
I haven't updated `neorocksTest` to use `nlua` yet.